CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.0.0
+

```

Comment:
I'd add a "spirit of this release on top", i.e. 

```
_A complete rewrite of the library, based on modern foundations._
```

Reply:
As you wish: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/666/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR11